tests/test_Explainer.py

```diff
@@ -122,31 +147,27 @@ def test_traversal_path_lcppn(data, request):
 
 @pytest.mark.skipif(not shap_installed, reason="shap not installed")
 @pytest.mark.parametrize("data", ["explainer_data", "explainer_data_no_root"])
-def test_traversal_path_lcpn(data, request):
+def test_traversal_path_lcpl(data, request):
     x_train, x_test, y_train = request.getfixturevalue(data)
     rfc = RandomForestClassifier()
-    lcpn = LocalClassifierPerNode(local_classifier=rfc, replace_classifiers=False)
+    lcpl = LocalClassifierPerLevel(local_classifier=rfc, replace_classifiers=False)
 
-    lcpn.fit(x_train, y_train)
-    explainer = Explainer(lcpn, data=x_train, mode="tree")
-    traversals = explainer._get_traversed_nodes_lcpn(x_test)
-    preds = lcpn.predict(x_test)
-
-    # Assert if predictions and traversals are of same length
+    lcpl.fit(x_train, y_train)
+    explainer = Explainer(lcpl, data=x_train, mode="tree")
+    traversals = explainer._get_traversed_nodes_lcpl(x_test)
+    preds = lcpl.predict(x_test)
     assert len(preds) == len(traversals)
-
-    # Assert if traversal path in predictions is same as the computed traversal path
     for i in range(len(x_test)):
         for j in range(len(traversals[i])):
-            label = traversals[i][j].split(lcpn.separator_)[-1]
+            label = traversals[i][j].split(lcpl.separator_)[-1]
```

Comment:
Please, don't delete the tests for the local classifier per node. Create new ones for the local classifier per level